### DISCUSSION TITLE
Findings don't display in Github Advanced Security UI correctly

### DISCUSSION THREAD

**Discussion by darkdatter** [aquasecurity/trivy#5804] [Bugs]
### Description

When utilizing the latest Github action for Trivy, the scan results don't display correctly under the Security tab, with Github Advanced Security enabled.

### Desired Behavior

Findings are displayed correctly, with pertinent details

### Actual Behavior

Finding is displayed in the security tab with formatting errors:

`Package: <resource>\nInstalled Version: 1.0\nVulnerability CVE-####-###\nSeverity: LOW\nFixed Version: 1.1\nLink:`

The summary section displays similar formatting errors, screenshot of both below:
![image](https://github.com/aquasecurity/trivy/assets/19212835/96fc3147-963c-402a-8f02-7617c167dbf0)

### Reproduction Steps

```bash
1. Utilize latest Github action for Trivy (aquasecurity/trivy-action@master)
2. Scan an image/output results to sarif with github/codeql-action/upload-sarif@v2
3. Review findings with formatting issues in Security tab
...
```

### Target

Container Image

### Scanner

Vulnerability

### Output Format

SARIF

### Mode

Standalone

### Debug Output

```bash
N/A
```

### Operating System

ubuntu-latest

### Version

```bash
aquasecurity/trivy-action@master
```

### Checklist

- [X] Run `trivy image --reset`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by DmitriyLewen** [comment]
Hello @darkdatter 
Thanks for your report!

We added escaping for sarif format - #5568

But looks like `github/codeql-action/upload-sarif@v2` doesn't work correctly with escaped characters.
I created https://github.com/orgs/community/discussions/80935 about this case.
Let's wait the answer.

Regards, Dmitriy

**Reply by DmitriyLewen** [reply]
Hello @darkdatter
We have reverted the changes to Sarif template - https://github.com/aquasecurity/trivy/pull/5897.
The next release will include these changes.

Thanks for your report!

Regards, Dmitriy
